src/v/resource_mgmt/memory_groups.h

```diff
@@ -17,12 +17,15 @@
  * Centralized unit for memory management.
  *
  * Works via a share system. First we subtract the amount of memory the user
- * decides to use for their WebAssembly functions. The remaining subsystems
- * are distributed memory via a share system.
+ * decides to reserve for their WebAssembly functions and for compaction
+ * buffers. The remaining subsystems are distributed memory via a share system.
```

Comment:
Should we update the memory_groups tests with this?

Reply:
Will do